### PR TITLE
Add GCP Next 18 joint Google, Waze, Netflix talk

### PR DESCRIPTION
**Pull request by t0mpson**
@duftler

### CODE REVIEW

**Comment by t0mpson** [comment]
@dorbin I think there's a bug where the presentations page is not linked at all from the main site. Any idea why?

**Comment by dorbin** [comment]
@t0mpson 
It's under Publications > Presentations, on the Community nav.

**Comment by t0mpson** [comment]
@dorbin Interesting. The menu is toggled off by default, so publications (and all other sub menus are effectively hidden by default) Is this a bug or by design?

**Comment by lwander** [comment]
We've collapsed all side menus across the site by default now because if they were expanded, they would take up too much space. Maybe there's another way we can make the "Presentations" page easier to find?

**Comment by dorbin** [comment]
Sorry, Tom. I'm OOO today. I'll look at this on Monday. 

We don't *have to* collapse navs on all pages by default, but if the tree on /community/ is expanded by default, it's just a little too long.

**Comment by dorbin** [comment]
@demobox 
Any thoughts on how we might make recent presentations more visible/findable?

File: community/presentations.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ redirect_from: /docs/presentations
 
 Conference and Meetup Presentations on Spinnaker
 
+### 
+<a href="https://youtu.be/PLNheBiWOGI" target="_blank">Large-Scale Continuous Delivery at Netflix and Waze Using Spinnaker (Cloud Next '18)</a> 
+Google Cloud Next ‘18 | July 24, 2019 | Andy Glover (Netflix), Matt Duftler (Google), Tom Feiner (Waze)
```
Comment (duftler) [inline community/presentations.md]: s/2019/2018

**Review by duftler — APPROVED** [review]
LGTM.

**Comment by dorbin** [comment]
#957

**Pull request merged**